### ISSUE TITLE
Undefined symbols "_freea" for architecture x86_64

### ISSUE THREAD

**Issue by elsifaka** [d12frosted/homebrew-emacs-plus#145]
When I do a `brew upgrade emacs-plus --fetch-HEAD`
It stops with:
```
Undefined symbols for architecture x86_64:
  "_freea", referenced from:
      _rpl_realpath in libgnu.a(canonicalize-lgpl.o)
```

I don't know how to open an issue upstream:
```
commit b9ac4f815ebaa1acb0d045fe9583f665efa6f628
Author: Daniel Colascione <dancol@dancol.org>
Date:   Sun Jun 23 18:19:08 2019 -0700

    Fix locating pdump by symlink

    * admin/merge-gnulib (GNULIB_MODULES): Add canonicalize-lgpl module
    * build-aux/config.guess, build-aux/gitlog-to-changelog,
    build-aux/update-copyright, lib/canonicalize-lgpl.c,
    lib/gnulib.mk.in, lib/malloca.c, lib/malloca.h, lib/pathmax.h,
    m4/canonicalize.m4, m4/double-slash-root.m4, m4/gnulib-comp.m4,
    m4/malloca.m4, my/pathmax.4: copy from GNUlib or regenerate from
    update
    * src/emacs.c: find dump by canonical path
```

Do I need to install something?

**Comment by pcr910303** [comment]
This bug is reported already upstream; Try recompiling in a few days :-)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by d12frosted** [comment]
Yeah, exactly as @pcr910303 says :) 

Actually, it's even a duplicate :)

**Comment by elsifaka** [comment]
:-) closing then, the beta suggestion does not work well yet :-)

**Comment by pcr910303** [comment]
@elsifaka Looks like this bug is fixed in commit 848ff94b45. Try rebuilding from HEAD :-)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by elsifaka** [comment]
@pcr910303 rebuilt and everything is fine 👍
